### PR TITLE
Change governance

### PR DESCRIPTION
**Pull request by RichiH**
Signed-off-by: Richard Hartmann <richih@richih.org>

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -25,11 +30,13 @@ Each project must have a [`MAINTAINERS.md`][maintainers] file with at least one
 
 ### Project lead
 
-The project lead may make decisions which hold weight equal to or higer than a supermajority vote. They may also veto any consensus or vote. While alive, the project lead can not be removed unless they choose to step down themselves.
+The project lead is voted for every two years by a simple majority vote. If no majority can be reached, the current project lead serves until a majority can be reached.
+
+The project lead can declare [rough consensus](#consensus), but voting is still open to override the consensus.
 
-Any decisions, vetos, or overrides under this provision must be clearly marked.
+The project lead gets one extra vote in votes other than project lead votes.
```
Comment (brian-brazil) [inline GOVERNANCE.md]: Comma after the first "votes" would be clearer.

**Review by brian-brazil — COMMENTED** [review]
:+1:

**Comment by robskillington** [comment]
Looks good to me 👍

File: GOVERNANCE.md (hunk 1 above)
Comment (RichiH) [reply GOVERNANCE.md]: Done, thanks.

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**